atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -253,16 +205,12 @@ const CreateCrashRecord = ({ client }) => {
           );
```

Comment:
`fatalityCountSum` and `susSeriousInjuryCountSum` can be removed bc they're not used 🚮